src/www/system_usermanager.php

```diff
@@ -635,7 +635,7 @@ function get_user_privdesc(& $user)
                   <tr>
                     <td><a id="help_for_email" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("E-Mail");?></td>
                     <td>
-                      <input name="email" type="text" value="<?= $pconfig['email'] ?>" />
+                      <input name="email" type="text" value="<?= $pconfig['email'] ?>" <?= $pconfig['scope'] == "system" || !empty($pconfig['user_dn']) ? "readonly=\"readonly\"" : "";?> />
```

Comment:
I don't think we should lock the email, if the other end doesn't return it, we likely get support questions about this.

Reply:
Agreed! We should probably have the same treatment for `descr` (fullname) as well, shouldn't we?

Reply:
that's debatable, it should be kept locked for system, but for ldap we might strip it off indeed

Reply:
Sure, so something like this works for `descr` (just removing check for empty `user_dn`)?

```
<input name="descr" type="text" value="<?=$pconfig['descr'];?>" <?= $pconfig['scope'] == "system" ? "readonly=\"readonly\"" : "";?> />
```